test/functional/p2p_v2_earlykeyresponse.py

```diff
@@ -59,6 +59,8 @@ def data_received(self, t):
         # check that data can be received on recvbuf only when mismatch from V1_PREFIX happens (send_net_magic = False)
         assert self.v2_state.can_data_be_received and not self.v2_state.send_net_magic
 
+    def on_open(self):
+        self.connection_made = True
```

Comment:
1d04d58: i think we should call it a different name - `self.connected` or something like that? i'm still getting a crash with the sleep statement because python is confusing `connection_made` variable and callback function.

Reply:
thanks, renamed to `connection_opened`